.github/workflows/submit.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         if: steps.check_submit.outputs.should_run != 'false' && steps.jtreg.outputs.cache-hit != 'true'
 
       - name: Build jtreg
-        run: sh make/build-all.sh ${JAVA_HOME}
+        run: sh make/build-all.sh ${JAVA_HOME_8_X64}
```

Comment:
What is this change?

Reply:
On ubuntu-20.04 the default Java installation is now set to 11, but jtreg still requires Java 8 for building.

Reply:
I see! Okay then.